kinto/core/resource/__init__.py

```diff
@@ -12,7 +12,7 @@
                                     HTTPNotFound, HTTPServiceUnavailable)
 
 from kinto.core import Service
-from kinto.core.errors import http_error, raise_invalid, send_alert, ERRORS
+from kinto.core.errors import http_error, raise_invalid, send_alert, ERRORS, request_GET
```

Comment:
I would have intuitively put in `utils` with all other request utilities. Any good argument for `errors`?